### PR TITLE
Return the stdout messages to improve test logging

### PR DESCRIPTION
**Pull request by akshita31**
dotnet test also prints the stdout messages. So added a property to `DotNetTestResult` to return an array of these messages.

omnisharp-vscode side : https://github.com/OmniSharp/omnisharp-vscode/pull/2343

Please review : @rchande @DustinCampbell

### CODE REVIEW

File: src/OmniSharp.DotNetTest/VSTestManager.cs [hunk 1]
```diff
@@ -240,7 +240,11 @@ public override RunTestResponse RunTest(string[] methodNames, string testFramewo
                     MethodName = testResult.TestCase.FullyQualifiedName,
                     Outcome = testResult.Outcome.ToString().ToLowerInvariant(),
                     ErrorMessage = testResult.ErrorMessage,
-                    ErrorStackTrace = testResult.ErrorStackTrace
+                    ErrorStackTrace = testResult.ErrorStackTrace,
+                    StdOutMessages = testResult.Messages
+                                    .Where(message => message.Category == TestResultMessage.StandardOutCategory)
```
Comment (rchande) [inline src/OmniSharp.DotNetTest/VSTestManager.cs]: Should we be picking up standard error here too?

Comment (rchande) [reply src/OmniSharp.DotNetTest/VSTestManager.cs]: And keeping it in its own category

File: src/OmniSharp.DotNetTest/Models/DotNetTestResult.cs [hunk 2]
```diff
@@ -6,5 +6,6 @@ public class DotNetTestResult
         public string Outcome { get; set; }
         public string ErrorMessage { get; set; }
         public string ErrorStackTrace { get; set; }
+        public string[] StdOutMessages {get; set; }
```
Comment (rchande) [inline src/OmniSharp.DotNetTest/Models/DotNetTestResult.cs]: `StandardOutput`

File: src/OmniSharp.DotNetTest/Models/DotNetTestResult.cs [hunk 3]
```diff
@@ -6,6 +6,7 @@ public class DotNetTestResult
         public string Outcome { get; set; }
         public string ErrorMessage { get; set; }
         public string ErrorStackTrace { get; set; }
-        public string[] StdOutMessages {get; set; }
+        public string[] StandardOutputMessages { get; set; }
```
Comment (rchande) [inline src/OmniSharp.DotNetTest/Models/DotNetTestResult.cs]: `StandardOutput` (no "messages")

File: src/OmniSharp.DotNetTest/VSTestManager.cs [hunk 4]
```diff
@@ -240,7 +240,12 @@ public override RunTestResponse RunTest(string[] methodNames, string testFramewo
                     MethodName = testResult.TestCase.FullyQualifiedName,
                     Outcome = testResult.Outcome.ToString().ToLowerInvariant(),
                     ErrorMessage = testResult.ErrorMessage,
-                    ErrorStackTrace = testResult.ErrorStackTrace
+                    ErrorStackTrace = testResult.ErrorStackTrace,
```
Comment (rchande) [inline src/OmniSharp.DotNetTest/VSTestManager.cs]: Is it possible to unit test this?

Comment (akshita31) [reply src/OmniSharp.DotNetTest/VSTestManager.cs]: Done!

**Pull request merged**